internal/web3ext/web3ext.go

```diff
@@ -577,6 +577,22 @@ web3._extend({
 			params: 3,
 			inputFormatter: [null, web3._extend.formatters.inputBlockNumberFormatter, null]
 		}),
+		new web3._extend.Method({
+			name: 'getLogs',
+			call: 'eth_getLogs',
+			params: 1,
+		}),
+		new web3._extend.Method({
+			name: 'call',
+			call: 'eth_call',
+			params: 4,
+			inputFormatter: [web3._extend.formatters.inputCallFormatter, web3._extend.formatters.inputDefaultBlockNumberFormatter, null, null],
+		}),
```

Comment:
how could these be missing all this time?

Reply:
Yeah, I'm also surprised. This is only for the interactive geth console, I don't recall ever using these two APIs on the console. Apparently these were added to upstream after we forked off.

Reply:
Ah, I thought this affected RPC as well.